docs/user-guide/advanced/replicate_evaluations.md

```diff
@@ -2,179 +2,156 @@
 
 To produce the evaluation results presented [here](../../index.md#evaluation-results), you can run *eva* with the settings below.
 
-Make sure to replace `<task>` in the commands below with `bach`, `crc`, `mhist` or `patch_camelyon`.
+The `.yaml` config files for the different benchmark datasets can be found [here](https://github.com/kaiko-ai/eva/tree/main/configs/vision).
+In the following commands, you need to replace `<task.yaml>` with the name of the config file you want to use.
 
-*Note that to run the commands below you will need to first download the data. [BACH](../../datasets/bach.md), [CRC](../../datasets/crc.md) and [PatchCamelyon](../../datasets/patch_camelyon.md) provide automatic download by setting the argument `download: true` (either modify the config-files or set the environment variable `DOWNLOAD=true`). In the case of MHIST you will need to download the data manually by following the instructions provided [here](../../datasets/mhist.md#download-and-preprocessing).*
+*Note: Some datasets provide automatic download by setting the argument `download: true` (either modify the `.yaml` config file or set the environment variable `DOWNLOAD=true`), while other datasets you need to download manually beforehand. Please review the instructions in the corresponding dataset [documentation](../../../datasets).*
 
-## DINO ViT-S16 (random weights)
+
+## Pathology FMs
+
+### DINO ViT-S16 (random weights)
 
 Evaluating the backbone with randomly initialized weights serves as a baseline to compare the pretrained FMs to an FM that produces embeddings without any prior learning on image tasks. To evaluate, run:
 
 ```
-PRETRAINED=false \
-EMBEDDINGS_ROOT="./data/embeddings/dino_vits16_random" \
-eva predict_fit --config configs/vision/dino_vit/offline/<task>.yaml
+MODEL_NAME="universal/vit_small_patch16_224_random" \
+EMBEDDINGS_ROOT="./data/embeddings/$MODEL_NAME" \
```

Comment:
this we don't need, right? Since we should have:
```
output_dir: &DATASET_EMBEDDINGS_ROOT ${oc.env:EMBEDDINGS_ROOT, ./data/embeddings}/${oc.env:MODEL_NAME, dino_vits16}/<task>
```
in all configs

Reply:
good catch. removed those and added a comment in the intruduction instead, pointing to `EMBEDDINGS_ROOT`